repo_health/check_github.py

```diff
@@ -168,7 +180,14 @@ async def check_settings(all_results, github_repo):
     results["last_push"] = github_repo.pushed_at
     repo_license = github_repo.license
     if repo_license is None:
-        results["license"] = None
+        if (
+            github_repo.name in repo_license_exemptions
+            and github_repo.owner.login
+            == repo_license_exemptions[github_repo.name]["owner"]
```

Comment:
I find this easier to read:
```suggestion
            and github_repo.owner.login == repo_license_exemptions[github_repo.name]["owner"]
```